webroot/img/flags/kiu.svg

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0"?>
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="30" height="20"><image  xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAB4AAAAUCAAAAAC/wNIYAAAA0ElEQVQoz63RoU7DYABF4SNmmkzOV0+BQmDq9gSTU9PTVRN9gCY1CBpsbQUWQ7Jkoq5vUIGpQFSQIJrsIOjq/hm48n7uXrwZ/shtWanW5avWZaN9WX7/Nq2YE6sVi5M+kOoJPlXvKa7cRuQGuY/ZXoI8JqwHg3yAxiAvgGOYYUPUhTkZE7YzN9CrrnkSc5YfNvB+5Q7O6riknmfZcTdOfFmxVyvoZu4inic2hX11jEic2ZTVMPHXIwBxp3jOCtUhyxpfsjfVsdpttsXwD3/fzA9S1rU+IQh+UgAAAABJRU5ErkJggg==" width="30" height="20"/></svg>
```

Comment:
If we keep the flag as is (just the ISO code `KIU`) I suggest to use a "real" svg image, for example:
```
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="30" height="20"><text x="2" y="18" style="font: bold 90% sans-serif">KIU</text></svg>
```